include/deal.II/numerics/vector_tools.templates.h

```diff
@@ -7082,7 +7082,7 @@ namespace VectorTools
         double my_values[3] = { mean_double.real(), mean_double.imag(), area };
         double global_values[3];
 
-        MPI_Allreduce (&my_values, &global_values, 3, MPI_DOUBLE,
+        MPI_Allreduce (my_values, global_values, 3, MPI_DOUBLE,
```

Comment:
I don't see how this could have worked (we overwrite the pointer ``global_values`` with garbage and then read from it). I assume we didn't test it...